contracts/pcv/curve/StableSwapOperatorV1.sol

```diff
@@ -83,6 +94,27 @@ contract StableSwapOperatorV1 is PCVDeposit {
         _usdt = IStableSwap3(_curve3pool).coins(2);
     }
 
+
+    /// @notice set the minimum ratio threshold for a valid reading of restistant balances
+    function setMinRatio(uint256 _minimumRatioThreshold) public onlyGovernor {
+        require(_minimumRatioThreshold < maximumRatioThreshold, "Min ratio must be less than max ratio");
```

Comment:
A good pattern for this is to have an internal method that gets called by both the governor setter and constructor

Reply:
@Joeysantoro fixed